docs/spec.md

```diff
@@ -2237,7 +2237,7 @@ contain the sizes of the slice for each dimension.
 
 More formally, `result[i0, ..., iR-1] = operand[j0, ..., jR-1]` where:
 
-* `jd = adjusted_start_indices[d][] + id`.
+* `jd = adjusted_start_indices[d] + id`.
```

Comment:
Given that `start_indices` is a vararg of 0-dimensional tensors, I imagine that `adjusted_start_indices` has the same type? If that's the case, then we need the second `[]` to turn a 0-dimensional tensor into a scalar, right?

Reply:
@burmako 
Thanks! I agree with why we should need.

Having noted that and assuming that `clamp` is same as `stablehlo.clamp`, there is still some confusion on the usage of `clamp` op  as  `stablehlo.clamp` which takes  arguments of the same shape, where as we have a var arg of `tensor<T>` (for `start_indices`) and `tensor<1xT>` (for `shape`). 


If you agree to that then we can add it to be done as part of #289.

Reply:
Sounds like a plan!